ukvm/ukvm_module_gdb.c

```diff
@@ -3,6 +3,9 @@
  *
  * This file is part of ukvm, a unikernel monitor.
  *
+ * Based on binutils-gdb/gdb/stubs/i386-stub.c, which is:
+ * Not copyrighted.
+ *
```

Comment:
Again, please keep the standard license header and move this into the next comment.